.travis.yml

```diff
@@ -25,7 +25,7 @@ install:
     if [[ "$SKIP_COVERAGE" != "1" ]]; then
       PYTEST_DJANGO_COVERAGE=1
       export PYTEST_ADDOPTS='--cov=pytest_django --cov=tests --cov=pytest_django_test --cov-report=term-missing:skip-covered'
-      export _PYTESTDJANGO_TOX_EXTRA_DEPS='https://github.com/pytest-dev/pytest-cov/archive/5735d9b34470508b0656d55d7c267fec2b3dbad4.tar.gz'
+      export _PYTESTDJANGO_TOX_EXTRA_DEPS='https://github.com/pytest-dev/pytest-cov/archive/58c3ef15075a24077a2395b0edf0a6952c956d33.tar.gz'
     else
       PYTEST_DJANGO_COVERAGE=0
     fi
```

Comment:
Failed due to non-absolute path to config: https://travis-ci.org/pytest-dev/pytest-django/builds/429211271#L587